CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ Framer Motion adheres to [Semantic Versioning](http://semver.org/).
 
 Undocumented APIs should be considered internal and may change without warning.
 
+## [9.0.3] 2023-02-14
+
+### Updated
+
+-   Refactored `motion` component features to load without React components.
+
+### Removed
+
+-   Fallback behavior for missing `IntersectionObserver`.
```

Comment:
Does this match our browser support matrix?

Reply:
I only know we already don't provide a fallback or polyfill on Framer sites